analysis/src/Event.cpp

```diff
@@ -52,6 +52,7 @@ PhysObjects Event::GetPhysObjects(EPhysObjType type, TH1D *cutFlowHist)
     return physObjects.at(type);
   }
   else if(type == kGoodGenPhoton)       return GetGoodGenPhotons();
+  else if(type == kPhotonInAcceptance)  return GetPhotonsInAcceptance();
```

Comment:
Suggestion (but feel free to leave it as is): one could use `GetPhotonsInAcceptance` method inside `GetGoodPhotons` and then there would be no need to check acceptance for each photon again. Something like this:

PhysObjects Event::GetGoodPhotons(){
  auto photonsInAcceptance = GetPhotonsInAcceptance();

  for(auto photon : photonsInAcceptance){
  // check only ID requirements, no need to check acceptance anymore

  }
  return ...
}

Reply:
Used this suggestion.

Reply:
I would prefer to keep it separate. The photonInAcceptance has no ID requirements only to check if it reconstructed or not and is in acceptance (pt, eta, ECAL crack and HEM)..

Reply:
This modification wouldn't change selection of photons! It would just remove some code duplication. But ok, let's keep it as is.